packages/devtools_app/lib/main.dart

```diff
@@ -16,8 +16,11 @@ import 'src/extension_points/extensions_external.dart';
 import 'src/globals.dart';
 import 'src/preferences.dart';
 import 'src/provider/riverpod_error_logger_observer.dart';
+import 'src/url_strategy/url_strategy.dart';
 
 void main() async {
+  usePathUrlStrategy();
```

Comment:
this will need to be applied to the google3 main file. I'll take an action item to do this when this change is rolled into g3.